crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -79,7 +80,15 @@ impl JsonRpcHandler {
         _block_id: BlockId,
         _contract_address: ContractAddressHex,
     ) -> RpcResult<ClassHashHex> {
-        Err(error::ApiError::ContractNotFound)
+        let parsed_address = _contract_address.0.try_into().unwrap(); // TODO better error handling
+
+        let starknet = self.api.starknet.read().await;
+        let state =
+            starknet.get_state_reader_at(&_block_id.into()).map_err(|_| ApiError::BlockNotFound)?;
```

Comment:
In other places in the code it's called like this:
`error::ApiError`
So we should probably decide on how to do it consistently?

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/50/commits/c14d56b31a6c8357e53efbe92519afa7d076448d